### ISSUE TITLE
Further Keras layer type compatibilities

### ISSUE THREAD

**Issue by RyanMetz** [SNL-NERL/Whetstone#1]
Hi! This looks absolutely brilliant. I have a specific Keras network I've already made and trained for NLP that I'd like to try with Whetstone. I was wondering about compatibility with layers not shown in the examples like GRU, embedding layers, and dropout. They aren't in layers.py either. Will those work with this?

**Comment by RyanMetz** [comment]
Just realized I was the first fork outside of your organization and I feel very cool.
